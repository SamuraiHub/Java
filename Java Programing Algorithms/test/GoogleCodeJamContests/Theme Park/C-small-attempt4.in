50
4 6 4
1 4 2 1
100 10 1
1
5 5 10
2 4 2 3 4 2 1 2 1 3
1 1 1
1
100 2 2
1 2
1000 100 10
4 3 9 1 2 5 6 9 1 10
1 10 10
4 3 4 1 2 5 6 9 1 10
1 10 9
1 1 1 1 1 1 1 1 1
1000 10 9
1 1 1 1 1 1 1 1 1
934 9 10
8 8 8 8 8 8 8 8 8 8
734 3 10
1 2 3 3 2 1 1 2 3 1
345 19 10
10 10 10 10 10 10 10 10 10 10
765 10 10
10 1 10 1 10 1 10 1 10 1
1000 100 10
10 10 10 10 10 10 10 10 10 10
734 20 10
1 2 3 4 5 6 7 8 9 10
631 16 10
10 9 8 7 6 5 4 3 2 1
4 9 7
3 7 2 2 4 9 8
2 5 8
5 2 4 3 1 3 4 1
4 14 7
8 4 5 1 10 4 7
3 12 5
9 6 10 6 3
407 6 9
2 5 2 1 4 4 5 3 5
890 9 9
4 1 6 1 6 2 1 2 8
556 13 10
4 10 5 6 10 6 4 9 4 9
651 10 9
6 2 5 6 6 7 5 8 2
7 21 9
5 2 8 3 10 7 6 2 2
7 20 9
6 9 5 5 2 10 1 8 1
6 22 10
4 8 7 7 7 8 7 3 9 4
9 20 9
10 2 6 2 4 6 10 2 10
95 15 8
7 7 5 8 6 5 8 8
26 27 9
9 8 7 9 9 9 9 6 5
115 16 9
8 6 8 7 8 5 7 8 7
191 46 9
7 6 6 6 7 5 8 8 8
477 33 8
8 7 8 9 5 6 8 8
95 16 10
8 8 6 6 7 7 7 9 5 8
452 78 10
7 8 6 6 7 9 5 6 9 6
590 74 9
8 9 9 6 7 9 9 8 6
838 91 10
7 8 6 5 6 9 9 6 7 7
374 28 8
9 8 6 6 7 5 9 6
278 43 8
6 8 9 9 6 7 5 6
776 67 8
8 7 7 9 9 7 9 8
978 17 6
8 7 8 5 9 7
628 87 7
9 5 6 5 8 5 5
695 58 8
7 5 5 7 6 7 9 8
731 97 9
9 9 6 9 6 5 6 9 7
615 56 9
7 5 9 8 6 9 5 9 8
219 48 5
6 9 9 9 6
393 79 5
8 7 5 5 7
777 100 6
7 9 7 5 6 7
741 25 8
7 9 6 5 8 9 9 7
196 79 5
8 9 6 6 6
