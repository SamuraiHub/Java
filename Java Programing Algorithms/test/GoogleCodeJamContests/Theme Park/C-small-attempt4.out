Case #1: 21
Case #2: 100
Case #3: 20
Case #4: 1
Case #5: 150
Case #6: 50000
Case #7: 7
Case #8: 9
Case #9: 9000
Case #10: 7472
Case #11: 1994
Case #12: 3450
Case #13: 4212
Case #14: 100000
Case #15: 13448
Case #16: 8669
Case #17: 27
Case #18: 7
Case #19: 39
Case #20: 25
Case #21: 1576
Case #22: 5518
Case #23: 6199
Case #24: 4371
Case #25: 125
Case #26: 119
Case #27: 115
Case #28: 156
Case #29: 1275
Case #30: 619
Case #31: 1637
Case #32: 8321
Case #33: 14074
Case #34: 1349
Case #35: 31188
Case #36: 41890
Case #37: 58660
Case #38: 8977
Case #39: 11120
Case #40: 49664
Case #41: 14344
Case #42: 27004
Case #43: 37530
Case #44: 48246
Case #45: 31569
Case #46: 8541
Case #47: 12576
Case #48: 31857
Case #49: 14827
Case #50: 6860
